index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porfolio project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <div class="header-section">
+<a href="#" class="logo">My Logo</a>
+<nav> <img src="images/Union.png" alt=""></nav>
+<!-- <a class="porfolio"href="#">Porfolio</a>
+<a href="#about" class="about">About</a>
+<a href="#contact" class="contact">Contact</a> -->
+</div>
+<section class="main-section">
+    <h1>I'm Therese TUYISABE<br> Glad to see you</h1>
+    <div class="primary text">
+<p class="text1">
+I am software developer!I can help you build a product feature or 
+website lookthrough sameof my work and experience!
+if you like what you see and have a project you need coded don't hestiate to contact me.
+</p>
+<p class="text2">Let's Connect</p></div>
+<div class="social-icon">
+<a href="https://twitter.com" class="twitter"> <img src="images/twitter.png" alt=""></a>
+<a href="https://linkedin.com" class="linkedin"><img src="images/Linkedin icon.png" alt=""></a>
+<a href="https://medium.com" class="medium"><img src="images/medium (2).png" alt=""></a>
+<a href="https://github.com" class="github"><img src="images/github.png" alt=""></a>
+<a href="https://angellist.com" class="angellist"><img src="images/angellist.png" alt=""></a>
```

Comment:
Kindly fix the paths to this images. They are not currently visible.
This is how you page looks
<img width="1222" alt="Screenshot 2023-01-19 at 15 35 17" src="https://user-images.githubusercontent.com/48546992/213446534-fb16af69-904e-4758-b9c7-d87f0ec89708.png">

This is the figma design:
<img width="381" alt="Screenshot 2023-01-19 at 09 40 33" src="https://user-images.githubusercontent.com/48546992/213446302-2df0282f-081c-4541-9ac8-94c58695b857.png">


Remember, The purpose of this project is to show that given a design you can replicate it. Also since it will serve as your portfolio, its important you can be reached via the social icons.